config/api.yml

```diff
@@ -703,6 +703,20 @@
     - :subcollection
     :methods: *70174834086080
     :klass: Dialog
+  :service_orders:
+    :description: Service Orders
+    :options:
+    - :collection
+    :methods: *70174834084700
+    :klass: ServiceOrder
+    :collection_actions:
+      :post:
+      - :name: create
+      - :name: edit
+      - :name: delete
+    :resource_actions:
+      :post:
+      - :name: edit
```

Comment:
@abellotti :+1: I'll add with a test

Re: identifiers, I don't believe they have been added yet. @AparnaKarve Can you confirm?